packages/config/src/types.ts

```diff
@@ -152,13 +153,7 @@ export interface ProjectLinks {
   socialMedia?: string[]
   rollupCodes?: string
 }
-
-export interface Badge {
-  id: string
-  type: string
-  name: string
-  description: string
-}
+export type Badge = (typeof badges)[number]
```

Comment:
Is this really needed? Every single other thing here could be defined without this. Can't we just add the filter field?